committee-code-of-conduct/charter.md

```diff
@@ -147,4 +147,12 @@ self-elected reason, they must notify the committee as well as the steering
 committee in writing. As a courtesy, such notifications should be given at least
 30 calendar days in advance of their departure.
 
+### Participation in the Kubernetes steering committee
+Participation in both of these committees represents a conflict of interest.
+If a current code of conduct committee member chooses to run for election for
+the Kubernetes steering committee, they are required to announce their 
+resignation from the code of conduct committee upon the announcement of their candidacy 
+for steering (not after the election takes place). Their resignation would then take 
+effect on the date of the election to allow notice time for a replacement member to be appointed.
```

Comment:
just to clarify: "on the date of the election", can we be extra-specific ... the date on which the election concludes or the date on which the voting begins? (I'd guess the former, to create a transition period leading up to potentially being on steering, but I'm not actually certain if that's the intent).

Reply:
Good catch! I'll clarify this